clar2wasm/src/standard/standard.wat

```diff
@@ -1770,4 +1788,7 @@
     (export "hash160-int" (func $hash160-int))
     (export "store-i32-be" (func $store-i32-be))
     (export "store-i64-be" (func $store-i64-be))
+    (export "not" (func $not))
+    (export "is-eq-int" (func $is-eq-int))
+    (export "xor" (func $bit-xor))
```

Comment:
You can delete this export too since the call is directly calling `bit-xor`.

Reply:
Ok... done! So, no implementation for `xor` at standard library.